config/owasp/suppressions.xml

```diff
@@ -19,13 +19,15 @@
       CVE-2018-10237
       CVE-2016-3094
       CVE-2022-45143
+      CVE-2022-40152
```

Comment:
What happened to the ordering that you made me suffer rearranging?!

Reply:
You're absolutely right.  Re-ordered the suppressions into ascending order.